test/easyconfigs/easyconfigs.py

```diff
@@ -926,6 +927,27 @@ def test_easyconfig_locations(self):
                     if not (dirpath.endswith('/easybuild/easyconfigs') and filenames == ['TEMPLATE.eb']):
                         self.assertTrue(False, "List of easyconfig files in %s is empty: %s" % (dirpath, filenames))
 
+    def test_easyconfig_name_clashes(self):
+        """Make sure there is not a name clash when all names are lowercase"""
+        topdir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+        names = defaultdict(list)
+        for (dirpath, _, _) in os.walk(topdir):
+            # ignore git/svn dirs & archived easyconfigs
+            if '/.git/' in dirpath or '/.svn/' in dirpath or '__archive__' in dirpath:
+                continue
+            dirpath_split = dirpath.replace(topdir, '').split(os.sep)
+            if len(dirpath_split) == 5:
+                name = dirpath_split[4]
+                names[name.lower()].append(name)
```

Comment:
maybe we can make the list of ignored dirs easier to maintain and also get rid of the `continue`  for the same price:
```suggestion
        # ignore git/svn dirs & archived easyconfigs
        ignore_dirs = ['.git', '.svn', '__archive__']
        for (dirpath, _, _) in os.walk(topdir):
            if not any('/%s' % d in dirpath for d in ignore_dirs):
                dirpath_split = dirpath.replace(topdir, '').split(os.sep)
                if len(dirpath_split) == 5:
                    name = dirpath_split[4]
                    names[name.lower()].append(name)
```